cl/scrapers/transformer_extractor_utils.py

```diff
@@ -0,0 +1,186 @@
+import json
+import os
+from tempfile import NamedTemporaryFile
+
+import requests
+from django.conf import settings
+from django.core import serializers
+
+
+def document_extract(path=None, file_content=None, do_ocr=False):
+    """Extract document content.
+
+    :param path:
+    :param file_content:
+    :param do_ocr:
+    :return:
+    """
+    service = "%s/%s" % (settings.BTE_URL, "extract_doc_content")
+    if path is not None:
+        with open(path, "rb") as file:
+            file_content = file.read()
+    elif file_content is None:
+        return {"err": "File not supplied"}
+
+    if path is not None:
+        file_name = os.path.basename(path)
+    else:
+        file_name = os.path.basename(file_content.name)
```

Comment:
I'm...confused what kind of object file_content is. If it's just a str of data, it wouldn't have a `name` property, right?